### PR TITLE
Station table seeded fr fr 

### PR DESCRIPTION
**Pull request by seoulstice**
70 stations now in Station table, previously only 19.

### CODE REVIEW

File: checklist.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-* [ ] Create a new station 
-* [ ] Update a station 
-* [ ] Delete a station (from either index or show)
-* [ ] Cannot save without name, dock count, city, installation date
+* [x] See all stations
+* [x] See one station
+* [x] Create a new station
+* [x] Update a station
+* [x] Delete a station (from either index or show)
+* [x] Cannot save without name, dock count, city, installation date
```
Comment (Ecksi) [inline checklist.md:13]: Nice, thanks for getting these.

File: db/seeds.rb [hunk 2]
```diff
@@ -1,9 +1,9 @@
 require 'csv'
 require './app/models/station'
 
-CSV.foreach('db/csv/station.csv', {headers: true, header_converters: :symbol}) do |row|
-  Station.create(name:              row[:name],
-                 city:              row[:city],
-                 dock_count:        row[:dock_count],
-                 installation_date: row[:installation_date])
-end
+  CSV.foreach('db/csv/station.csv', {headers: true, header_converters: :symbol, converters: :numeric}) do |row|
```
Comment (Ecksi) [inline db/seeds.rb]: What does converters: :numeric do?

**Review by Ecksi — APPROVED** [review]
Looks good!

File: db/seeds.rb (hunk 2 above)
Comment (seoulstice) [reply db/seeds.rb]: It converts strings that look like numbers in the csv file to actual numbers

Comment (seoulstice) [reply db/seeds.rb]: like the lat/long and the dock_count

Comment (Maxscores) [reply db/seeds.rb]: That's pretty cool, nice work getting the date setup

File: db/seeds.rb [hunk 3]
```diff
@@ -1,2 +1,9 @@
 require 'csv'
 require './app/models/station'
+
+  CSV.foreach('db/csv/station.csv', {headers: true, header_converters: :symbol, converters: :numeric}) do |row|
+    Station.create!(name:             row[:name],
+                   city:              row[:city],
+                   dock_count:        row[:dock_count],
+                   installation_date: Date.strptime(row[:installation_date], '%m/%e/%Y'))
```
Comment (Maxscores) [inline db/seeds.rb:8]: nice use of %e here. Had no idea that it existed.

**Pull request merged**